games-strategy/hedgewars/hedgewars-0.9.25.ebuild

```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+CMAKE_MAKEFILE_GENERATOR=emake
+inherit cmake-utils desktop xdg-utils
```

Comment:
It does not appear that this ebuild actually uses any functions from `desktop.eclass`, correct me
if I'm wrong.

Reply:
I removed desktop eclass from the ebuild.